.pa11yci

```diff
@@ -0,0 +1,164 @@
+                "wait for path to be /requests/workshop_request/2/"
+            ]
+        },
+        "http://127.0.0.1:8000/requests/workshop_request/1/edit/",
+        "http://127.0.0.1:8000/requests/workshop_request/1/accept_event/",
+        "http://127.0.0.1:8000/fiscal/membership/1/",
+        "http://127.0.0.1:8000/fiscal/membership/1/members/",
+        "http://127.0.0.1:8000/recruitment/process/add/1/",
+        "http://127.0.0.1:8000/recruitment/process/1/add-signup",
+        "http://127.0.0.1:8000/recruitment/process/1/",
+        "http://127.0.0.1:8000/recruitment/processes/",
+        "http://127.0.0.1:8000/reports/duplicate_persons/",
+        "http://127.0.0.1:8000/reports/duplicate_training_requests/",
+        "http://127.0.0.1:8000/reports/instructor_issues/",
+        "http://127.0.0.1:8000/reports/membership_trainings_stats/",
+        "http://127.0.0.1:8000/reports/workshop_issues/",
+        {
+            "url": "http://127.0.0.1:8000/trainings/trainees/",
+            "actions": [
+                "wait for element table to be visible"
+            ]
+        },
+        {
+            "url": "http://127.0.0.1:8000/trainings/trainings/",
+            "actions": [
+                "wait for element table to be visible"
+            ]
+        },
+        "http://127.0.0.1:8000/trainings/training_progress/1/edit/",
+        "http://127.0.0.1:8000/workshops/airport/ALC/",
+        {
+            "url": "http://127.0.0.1:8000/workshops/badges/",
+            "actions": [
+                "wait for element table to be visible"
+            ]
+        },
+        {
+            "url": "http://127.0.0.1:8000/workshops/events/",
+            "actions": [
+                "wait for element table to be visible"
+            ]
+        },
+        {
+            "url": "http://127.0.0.1:8000/workshops/persons/",
+            "actions": [
+                "wait for element table to be visible"
+            ]
+        },
+        {
+            "url": "http://127.0.0.1:8000/workshops/tasks/",
+            "actions": [
+                "wait for element table to be visible"
+            ]
+        },
+        "http://127.0.0.1:8000/workshops/badge/instructor/",
+        "http://127.0.0.1:8000/workshops/log/",
+        "http://127.0.0.1:8000/workshops/events/add",
+        "http://127.0.0.1:8000/workshops/events/merge/?event_a=1&event_b=2",
+        "http://127.0.0.1:8000/workshops/events/metadata_changed/",
+        "http://127.0.0.1:8000/workshops/version/1/",
+        "http://127.0.0.1:8000/workshops/persons/bulk_upload/",
+        "http://127.0.0.1:8000/workshops/persons/add/",
+        "http://127.0.0.1:8000/workshops/persons/merge/?person_a=2&person_b=3",
+        "http://127.0.0.1:8000/workshops/person/1/",
+        "http://127.0.0.1:8000/workshops/person/1/edit/",
+        "http://127.0.0.1:8000/workshops/task/1/",
+        "http://127.0.0.1:8000/workshops/workshop_staff/",
+        "http://127.0.0.1:8000/account/logout/"
```

Comment:
I assume pa11y doesn't support "HOST" setting and we'll have to maintain this list of URLs?

Reply:
Yes, we have to maintain it. pa11y [can use a sitemap](https://github.com/pa11y/pa11y-ci#sitemaps), though we would still need to manually specify URLs where we use custom actions. (Lighthouse doesn't support using a sitemap)
Django does support [creating sitemaps](https://docs.djangoproject.com/en/4.2/ref/contrib/sitemaps/), but it doesn't seem particularly straightforward.